lib/src/syn/v1/stmt/define/field.rs

```diff
@@ -141,6 +141,6 @@ fn field_comment(i: &str) -> IResult<&str, DefineFieldOption> {
 
 fn field_permissions(i: &str) -> IResult<&str, DefineFieldOption> {
 	let (i, _) = shouldbespace(i)?;
-	let (i, v) = permissions(i)?;
+	let (i, v) = permissions(i, None)?;
```

Comment:
Because this could be confused with `Permission::None`, even though it's actually `Option<Permission>` and therefore `Permission::Full`:

```suggestion
	let (i, v) = permissions(i, Permission::Full)?;
```